projects/ffmpeg/run_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Usage:
+#   bash run_test.sh list     - Lists all available tests
```

Comment:
maybe we can just start simple with running all tests? 

it's hard to tell which tests are affected and need to be run after a change, so we probably want to run them all to begin with.